great_expectations/core/usage_statistics/anonymizers/expectation_suite_anonymizer.py

```diff
@@ -1,10 +1,21 @@
 from great_expectations.core.usage_statistics.anonymizers.anonymizer import Anonymizer
 from great_expectations.dataset import Dataset
+from great_expectations.expectations.registry import (
+    list_registered_expectation_implementations,
+)
 
-GE_EXPECTATION_TYPES = [
+v2_batchkwargs_api_supported_expectation_types = [
     el for el in Dataset.__dict__.keys() if el.startswith("expect_")
 ]
 
+v3_batchrequest_api_supported_expectation_types = (
+    list_registered_expectation_implementations()
```

Comment:
@anthonyburdi Possibly not for this PR, but why do we have the `if` statement in `list_registered_expectation_implementations()`?.. :)

Reply:
Hmm good question. It looks like we can use `expectation_root` to filter the returned expectation implementations to only those that are subclasses of the expectation type we pass as `expectation_root`. I don't currently see it used yet in our codebase.

Reply:
... but the action is the same for both parts of the predicate...

Reply:
Good point. But the condition is different. For example, if you don't pass an `expectation_root` then all `expectation_name`s will be returned. If you do pass an `expectation_root` then only if the expectation behind the `expectation_name` is a subclass of `expectation_root` will it be appended and returned. So if you only want expectations that are derived from `ExpectColumnValuesToNotBeNull` then pass that as the `expectation_root`. Does that make sense?